### PR TITLE
Timestamps on the step page

### PR DESCRIPTION
**Pull request by novemberkilo**
This builds on https://github.com/ocurrent/ocaml-ci/pull/516 to use `Timedesc` to write out ISO8601 timestamps in the local timezone.

Screen Shot 2022-08-25 at 4 32 57 pm

### CODE REVIEW

File: web-ui/view/github.ml [hunk 1]
```diff
@@ -327,6 +351,9 @@ let show_step ~org ~repo ~refs ~hash ~jobs ~variant ~job ~status ~csrf_token
         ]
       else []
     in
+    let step_info =
+      List.find_opt (fun (j : Client.job_info) -> j.variant = variant) jobs
+    in
```
Comment (maiste) [inline web-ui/view/github.ml:356]: Just a suggestion (to follow ocaml.org code guideline) you can choose to apply or not:
```suggestion
    let step_info =
      let filter (j : Client.job_info) = j.variant = variant in
      List.find_opt filter jobs
    in
```

**Review by maiste — APPROVED** [review]
LGTM :+1:

File: web-ui/view/github.ml [hunk 2]
```diff
@@ -175,6 +175,30 @@ let list_refs ~org ~repo ~refs =
       refs_v ~org ~repo ~refs;
     ]
 
+let show_timestamps step_info =
+  let to_iso8601 (tt : float) =
+    let ts = Timedesc.of_timestamp_float_s tt in
+    Timedesc.to_iso8601 @@ Option.get ts
```
[the remark this answers is not in the document]
Comment (novemberkilo) [reply web-ui/view/github.ml:181]: I think we want to fail hard here if we end up with a tt that cannot be turned into a timestamp.

**Comment by novemberkilo** [comment]
Closing this in favour of #522

**Pull request closed without merging**